demo/loadmore.html

```diff
@@ -45,7 +45,6 @@
   </script>
   <script type="module">
```

Comment:
This should probably be moved to the head (it seems nicer to import things before they're used, just for code readability) and then can be simplified to `<script type="module" src="./util/confirm-features.mjs"></script>`

Reply:
done.